edge-modules/api-proxy-module/src/main.rs

```diff
@@ -106,10 +106,7 @@ pub fn nginx_controller_start(
         //Wait for configuration to be ready.
         notify_config_reload_api_proxy.notified().await;
 
-        //Wait for the trust bundle.
-        notify_cert_reload_api_proxy.notified().await;
-
-        //Wait for the server cert and private key.
+        //Wait for the trust bundle and server cert
```

Comment:
Can this fix be simplified by using different instances of Notify for trust_bundle and server_cert?